test/langtools/tools/javac/diags/examples/SelectorTypeNotAllowed.java

```diff
@@ -21,7 +21,8 @@
  * questions.
  */
 
-// key: compiler.err.selector.type.not.allowed
+// key: compiler.err.preview.feature.disabled.plural
```

Comment:
The key does not exist any, per my understanding. I would suggest to simply delete the file.